.ci/E2E-tests/execute.sh

```diff
@@ -45,15 +49,29 @@ export HOST_HOSTNAME=$(hostname)
 cd docker
 #just pull everything else than artemis-app as we build it later either way
 if [ "$TEST_FRAMEWORK" = "playwright" ]; then
-  echo "Building for playwright"
-  docker compose -f $COMPOSE_FILE pull artemis-playwright $CONFIGURATION nginx
-  docker compose -f $COMPOSE_FILE build --build-arg WAR_FILE_STAGE=external_builder --no-cache --pull artemis-app
-  docker compose -f $COMPOSE_FILE up --exit-code-from artemis-playwright
+  if [ "$CONFIGURATION" = "multi-node" ]; then
+    echo "Building for playwright"
```

Comment:
Not quite sure about the usage, but I guess it might be useful to print in which branch we are?

```suggestion
    echo "Building for playwright (multi-node)"
```